src/ensembl/src/shared/helpers/numberFormater.test.ts

```diff
@@ -0,0 +1,31 @@
+import { getCommaSeparatedNumber } from './numberFormatter';
+
+describe('getCommaSeparatedNumber', () => {
+  it('returns 1234 for the input 1234', () => {
+    expect(getCommaSeparatedNumber(1234)).toBe('1,234');
+  });
+
+  it('returns 12,345 for the input 12345', () => {
+    expect(getCommaSeparatedNumber(12345)).toBe('12,345');
+  });
+
+  it('returns 123,456 for the input 123456', () => {
+    expect(getCommaSeparatedNumber(123456)).toBe('123,456');
+  });
+
+  it('returns 1,234,567 for the input 1234567', () => {
+    expect(getCommaSeparatedNumber(1234567)).toBe('1,234,567');
+  });
+
+  it('returns -1,234,567 for the input -1234567', () => {
+    expect(getCommaSeparatedNumber(-1234567)).toBe('-1,234,567');
+  });
+
+  it('returns 0 if the inout is an empty string', () => {
+    expect(getCommaSeparatedNumber('')).toBe('0');
+  });
+
+  it('returns 0 if the inout is undefined', () => {
```

Comment:
this and previous: inout -> input. Also, should  this actually work for empty strings or undefineds? Shouldn't the function just receive type number?

Reply:
I was thinking what if numbers are passed in as a string, but I can see that it can also accept inputs like 'hello', which is not good.
I'll change it to only accept numbers.